src/FlightBooking.Core/Entities/Location.cs

```diff
@@ -0,0 +1,17 @@
+﻿using FlightBookingApp.Core.Common;
+
+namespace FlightBookingApp.Core.Entities
+{
+    public class Location : BaseAuditableEntity
+    {
+        public string? City { get; set; }
+
+        public string? State { get; set; }
+
+        public string? Country { get; set; }
+
+        public string? Address { get; set; }
+
+        public Flight? Flight { get; set; }
```

Comment:
Location should not be unique for each flight, so such relationship is incorrect. Relationship between flight and location must be many to one, cause location can have many flights, but flight can have only one destination/departure location